src/vorta/views/repo_add_dialog.py

```diff
@@ -107,7 +107,7 @@ def validate(self):
             return False
 
         if len(self.values['repo_name']) > 64:
-            self._set_status(self.tr('Repository name must be less than 64 characters.'))
```

Comment:
Faulty wording, should say "less than 65 characters" or "no more than 64 characters"